app/components/spin-button.js

```diff
@@ -80,7 +80,7 @@ export default Ember.Component.extend({
     }
   },
 
-  inFlightDidChange: function() {
+  inFlightDidChange: Ember.observer(function() {
```

Comment:
Nice. Could you make this just `observer(fn..)` and destructure `Ember.observer` above.